app/src/main/res/values/strings.xml

```diff
@@ -617,7 +617,12 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_railway_crossing_barrier_none">It\'s not</string>
     <string name="action_deselect_all">Deselect all</string>
     <string name="quest_buildingType_retail_description">including buildings for restaurants, cafés, …</string>
-
+    <string name="quest_maxweight_title">"What is the weight limit here?"</string>
+    <string name="quest_maxweight_answer_noSign">There is no sign</string>
+    <string name="quest_maxweight_unusualInput_confirmation_description">This weight looks implausible. Are you sure that it is correct?</string>
+    <string name="quest_maxweight_answer_other_sign">There is some other weight limit sign…</string>
+    <string name="quest_maxweight_answer_sign_with_exceptions">There are exceptions…</string>
+    <string name="quest_maxweight_unsupported_sign_request_photo">Would you like to leave a note? A note with a photo will enable other mappers to handle it. Common signs will also be added to StreetComplete, allowing a direct input.</string>
```

Comment:
```suggestion
    <string name="quest_maxweight_unsupported_sign_request_photo">Would you like to leave a note? A note with a photo will enable other mappers to take care of it.</string>
```

I find this note about that StreetComplete will improve in the future problematic. This would also be translated to all the other languages and then later re-translated when the text is changed.